### DISCUSSION TITLE
Disable backspace altering the model in QSelect

### DISCUSSION THREAD

**Discussion by turigeza** [quasarframework/quasar#9681] [General - Components / Directives / etc] [answered]
From the docs 

https://quasar.dev/vue-components/select#keyboard-navigation
> pressing BACKSPACE will remove the last option from the selection

I would like to disable that. Any ideas ? I tried to capture the native input event but it doesn't fire once the input element is empty. 

Thank you for helping,

**Comment by ontwikkelfabriek** [comment, marked as answer]
@turigeza 

Here's a codepen, it's a hacky solution, because it's **hardcoded** in the q-select with no way to disable( as far as I know)
https://codepen.io/ontwikkelfabriek/pen/NWpeeLK

If you want to take a look at the source code of q-select code that removes an item on `backspace` keypress (btw I found this hack by looking at this code):
https://github.com/quasarframework/quasar/blob/dev/ui/src/components/select/QSelect.js#L682

Here's another codepen that's not dependent on the `hack`:
https://codepen.io/ontwikkelfabriek/pen/RwpEvwd

**Comment by turigeza** [comment]
@ontwikkelfabriek Thank you for your help. : ) I went with the last one.
